frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -1,4 +1,44 @@
-# This file consists of guide lines to create automation scripts
+# Getting Started
+
+## Directory Structure
+
+Folder structure of cypress cucumber framework
+
+```
+frontend/packages/dev-console/integration-tests/
+├── support              <--- cypress cucumber support configurations
+|   ├── commands         <--- add commands to Cypress 'cy.' global, other support configurations
+|   |   └── index.ts
```

Comment:
There is also an `app.ts` under `support/commands`

Reply:
updated